lib/peddler/client.rb

```diff
@@ -100,6 +108,7 @@ def operation(action = nil)
     end
 
     def run
+      operation.store('Version', version) unless version.to_s == ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.